examples/kubernetes/samba-ad-server-deployment-local.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: samba-ad-server
+  labels:
+    app: samba
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: samba-ad
+  template:
+    metadata:
+      labels:
+        app: samba-ad
+    spec:
+      containers:
+      - name: samba
+        image: samba-ad-container:latest
+        imagePullPolicy: Never
```

Comment:
I'm confused as to how this "samba-ad-server-deployment-local.yml" is supposed to differ from the other one. Unless I am missing something the main difference is the imagePullPolicy, but I don't understand what that's meant to accomplish.

Reply:
The main point was using the locally built image instead of pulling the one from quay.io.
After some googling it seemed for it to work, I had to change the pull policy to Never. 🤷

Reply:
If we can get away with just one deployment - fine!

Reply:
OK, my thoughts are that since this is an example file, we can simply do one file, perhaps with comments added to explain some of the common things a person might want to change. 

FWIW, I usually just change the image to reference a container registry I run on my machine or my own repo on a public container registry. The former has the added advantage that its really much faster, but has the downside you must to extra configuration steps before it will work.

Reply:
updated to only have one yaml. the one with the local image.